llvm/lib/SYCLLowerIR/PrepareSYCLNativeCPU.cpp

```diff
@@ -276,107 +283,31 @@ PreservedAnalyses PrepareSYCLNativeCPUPass::run(Module &M,
   // Then we iterate over all the supported builtins, find their uses and
   // replace them with calls to our Native CPU functions.
   for (auto &Entry : BuiltinNamesMap) {
```

Comment:
Could `Entry` be const? I think it's not modified anywhere.

Reply:
Done, thanks for spotting it